omni/omni.py

```
@@ -0,0 +1,1085 @@
+                headers = {
+                    "x-omni": h
+                }
+                session = self.session
+                if session is None or session.closed:
+                    session = aiohttp.ClientSession()
+                async with session.post(
+                    "https://automator.beehive.systems/api/v1/webhooks/hj05HelXPKgXZQEAUWf7T",
+                    json=payload,
+                    headers=headers,
+                    timeout=10
+                ) as resp:
```

Ruff (0.8.2)

Local variable `resp` is assigned to but never used

Remove assignment to unused variable `resp`

(F841)